Cargo.toml

```diff
@@ -43,3 +43,4 @@ lto = true
 # https://github.com/rust-lang/cargo/issues/6988
 [profile.bench]
 debug = true
+debug-assertions = false
```

Comment:
Note: I made this change because when running Valgrind, debug-assertions was producing a lot of noise.  This change also disables debug-assertions for the memory benchmarks, which is probably a good thing.